src/main/java/com/dreammaster/gthandler/recipes/AssemblerRecipes.java

```diff
@@ -931,14 +931,12 @@ public void run() {
                 GT_ModHandler.getModItem(IronTanks.ID, "goldDiamondUpgrade", 1L, 0),
                 600,
                 120);
-
-        GT_Values.RA.addAssemblerRecipe(
-                new ItemStack[] { CustomItemList.RawLapotronCrystal.get(1L),
-                        GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Advanced, 2L) },
-                GT_Values.NF,
-                GT_ModHandler.getIC2Item("lapotronCrystal", 1L),
-                600,
-                1024);
+        GT_Values.RA.stdBuilder()
+                .itemInputs(
+                        CustomItemList.RawLapotronCrystal.get(1L),
+                        GT_OreDictUnificator.get(OrePrefixes.circuit, Materials.Advanced, 2L))
+                .itemOutputs(GT_ModHandler.getIC2Item("lapotronCrystal", 1L, 26)).noFluidInputs().noFluidOutputs()
+                .duration(30 * SECONDS).eut(TierEU.RECIPE_EV / 2).addTo(sAssemblerRecipes);
```

Comment:
TierEU.RECIPE_EV is corresponding to 1920 EU/t, so dividing it by 2 it's 960 EU/t, while the previous recipe had 1024 EU/t. Is it intended? Also, if anything, i'd rather make it run at 1920 EU/t with a 320 seconds duration (same power used in total) What do you think?

Reply:
when this last came up I was corrected for using 1024 instead of 960. cant please everyone :)

I really dont care. I will make it 16 seconds at 15/16A then (not 320 seconds lul)

Reply:
oh sorry, did a silly mistake, i meant 320 ticks, so 16s yeah